kodilla-hibernate/src/main/java/com/kodilla/hibernate/task/TaskListDao.java

```diff
@@ -0,0 +1,14 @@
+package com.kodilla.hibernate.task;
+
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Transactional
```

Comment:
https://stackoverflow.com/questions/5551541/where-to-put-transactional-in-interface-specification-or-implementation